test/unit-tests/common/test_common.py

```diff
@@ -13,6 +13,8 @@ class TestConfluenceCommon(unittest.TestCase):
     @classmethod
     def setUpClass(self):
         self.config = _.prepareConfiguration()
+        self.config['extensions'].append('sphinx.ext.todo')
```

Comment:
Since this PR is targeting support for an extension (`sphinx.ext.todo`), I would recommend we create a separate unit testing folder to help separate base features vs. extension features. Instead of placing these tests in `test/unit-tests/common`, it maybe better to place them into `test/unit-tests/ext-todo` instead.